security/netbird/src/opnsense/mvc/app/controllers/OPNsense/Netbird/Api/ServiceController.php

```diff
@@ -4,6 +4,7 @@
  * Copyright (C) 2025 Ralph Moser, PJ Monitoring GmbH
  * Copyright (C) 2025 squared GmbH
  * Copyright (C) 2025 Christopher Linn, BackendMedia IT-Services GmbH
+ * Copyright (C) 2025 NetBird GmbH
```

Comment:
```suggestion
 * Copyright (C) 2025 NetBird GmbH
```

the whole file was rewritten. I thought about this for a while and for each file you rewrite completely we cannot assert anyone else worked on them.